frontend/public/components/marketplace/modal-overlay.jsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+
+import Button from 'patternfly-react/dist/esm/components/Button/Button';
+import Modal from 'patternfly-react/dist/esm/components/Modal/Modal';
+import CatalogItemHeader from 'patternfly-react-extensions/dist/esm/components/CatalogItemHeader/CatalogItemHeader';
+import PropertiesSidePanel from 'patternfly-react-extensions/dist/esm/components/PropertiesSidePanel/PropertiesSidePanel';
+import PropertyItem from 'patternfly-react-extensions/dist/esm/components/PropertiesSidePanel/PropertyItem';
+
+class MarketplaceModalOverlay extends React.Component {
+  constructor(props){
+    super(props);
+  }
```

Comment:
I think constructor might be unnecessary here, since state isn't being set.

Reply:
You don't necessarily need a constructor to set state, either.  https://daveceddia.com/where-initialize-state-react/

Reply:
How would ``` this.props ``` be defined if constructor is removed?

Reply:
`props` are inherited

Reply:
`props` will be set by default without a constructor, I believe.

Reply:
Ok will remove it then